courseraprogramming/commands/upload.py

```diff
@@ -341,11 +341,16 @@ def parser(subparsers):
 
     parser_upload.add_argument(
         'course',
-        help='The course id to associate the grader.')
+        help='The course id to associate the grader. The course id is a '
```

Comment:
@saeta Would it make sense to allow to add in a new parameter course_slug (to preserve backwards compatibility) and have courseraprogramming call the catalog API to resolve the courseId rather than having instructors call it?